config/routes.rb

```diff
@@ -33,4 +33,5 @@
   post '/favorites', to: 'adoption_application#create'
   get '/adoption_applications/:application_id', to: 'adoption_application#show'
   get '/pets/:pet_id/applications', to: 'pet_applications#index'
+  patch '/pets/:pet_id/applications/:application_id', to: 'pet_applications#update'
```

Comment:
I'm not confident this is RESTful, but it was the best way I could see to pass both these ids  as params to the controller. Do you have any suggestions on a better way to accomplish this?

Reply:
Not at the moment. Perhaps we can look at this together.

Reply:
I think query params might be the way to go? I put this in the README as something we would like to explore in the future.

Reply:
That is something I'm not very familiar with, and will take a look at how it's done. Thank you.